index.html

```diff
@@ -298,5 +314,13 @@ <h2>note</h2>
       <p style="margin-bottom: 0;">Last updated: 2017-01-04</p>
     </div>
   </div>
+  <script type="text/javascript">
+    (function() {
+      var urlParams = new URLSearchParams(window.location.search);
+      if(urlParams.has('w')) {
+        document.getElementById("ipa-input").value = urlParams.get('w');
```

Comment:
Would it be possible to add some input validation? Tricking the user into pasting in some large or unexpected input may take a bit more effort than having them click a link.

Reply:
It's a good point. I added a little validation, check the word:

- not included html tag
- less than 200 characters